client/card-readers/settings/sections/contacts-details.js

```diff
@@ -69,21 +85,28 @@ const ContactDetailsSection = ( { setSaveDisabled } ) => {
 				value={ accountBusinessSupportEmail }
 				onChange={ setAccountBusinessSupportEmail }
 				type="email"
-				onBlur={ updateSaveButtonAvailability }
 			/>
 			{ businessSupportPhoneErrorMessage && (
 				<Notice status="error" isDismissible={ false }>
 					<span>{ businessSupportPhoneErrorMessage }</span>
 				</Notice>
 			) }
-			<TextControl
-				className="card-readers-business-phone-input"
-				label={ __( 'Support phone number', 'woocommerce-payments' ) }
-				value={ accountBusinessSupportPhone }
-				onChange={ setAccountBusinessSupportPhone }
-				type="tel"
-				onBlur={ updateSaveButtonAvailability }
-			/>
+			<div className="card-readers-business-phone-input">
+				<span>
```

Comment:
I guess this could be `label`. I noticed there is less margin between text and the input:

<img width="411" alt="Screen Shot 2022-07-13 at 10 44 27" src="https://user-images.githubusercontent.com/1553182/178691881-62f0e872-bdfc-4d68-8e7b-77000265d221.png">

Reply:
Oh, missed this!

I wanted to hard-code a margin, but then made a quick search and found the `BaseControl` exposed from `@wordpress/components`. I ended up wrapping the component with it https://github.com/Automattic/woocommerce-payments/pull/4355/commits/b331184c9a2f5367faf0668906bffcf4b48362ad